src/index.html

```diff
@@ -2,11 +2,488 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+          href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap" 
+          rel="stylesheet"
+    >
   </head>
+
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+
+      <a href="index.html" class="logo">
+        <img src="\images\logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link is-active">Apple</a>
```

Comment:
there shouldn’t be empty line between child and parent element

Reply:
fix it for the whole page

Reply:
Done